test/helper/routeHelper.js

```diff
@@ -79,6 +79,7 @@ async function getRecipeRoute({ app }, token) {
 }
 
 async function getRecipeByIdRoute({ app }, id, token) {
+  console.log({ id })
```

Comment:
Should we be logging this?

Reply:
we should not, I was just troubleshooting

Reply:
to be fair it does not matter as it's test helper, but you are right this should not be there